.github/workflows/ci.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: CI
+
+on: [push, pull_request]
```

Comment:
Just a note (to both @bharling and reviewers) - PR #25 _also_ adds similar CI, but does so a bit differently ensuring CI isn't invoked redundantly on _both_ PR and non-master push events:

```yaml
on:
  pull_request:
  push:
    branches:
      - master
```